### PR TITLE
OPENTOK-48553: Update all samples to latest Opentok SDK version

### PR DESCRIPTION
**Pull request by goncalocostamendes**
Update Basic Video Chat to latest Opentok SDK version

https://jira.vonage.com/browse/OPENTOK-48553

### CODE REVIEW

**Comment by goncalocostamendes** [comment]
@v-kpheng similar case here as Android. I can start adding the version of the latest SDK version to more samples, specially since only changing the sdk version is not enough. Also the minimun target version has to be increased to 12.

**Comment by v-kpheng** [comment]
> @v-kpheng similar case here as Android. I can start adding the version of the latest SDK version to more samples, specially since only changing the sdk version is not enough. Also the minimun target version has to be increased to 12.

That's a good idea :-)
Yeah, let's change all of them to use the latest version, please, if possible 🙏 🤞

**Review by pvenum — APPROVED** [review]

File: Broadcast-Ext/Broadcast-Ext.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -731,19 +765,23 @@
 				SDKROOT = iphoneos;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VALIDATE_PRODUCT = YES;
+				VALIDATE_WORKSPACE = YES;
```
Comment (goncalocostamendes) [inline Broadcast-Ext/Broadcast-Ext.xcodeproj/project.pbxproj:768]: @IGitGotIt @jvalli please provide feedback. I was facing the following issue 

> /Users/gmendes/samples/opentok-ios-sdk-samples/Broadcast-Ext/Broadcast-Ext.xcodeproj: error: Building for iOS Simulator, but the linked and embedded framework 'OpenTok.framework' was built for iOS + iOS Simulator. (in target 'Broadcast-Ext' from project 'Broadcast-Ext')
/Users/gmendes/samples/opentok-ios-sdk-samples/Broadcast-Ext/Broadcast-Ext.xcodeproj: error: The linked framework 'Pods_Broadcast_Ext.framework' is missing one or more architectures required by this target: i386. (in target 'Broadcast-Ext' from project 'Broadcast-Ext')

, that was solved by allowing workspace validation. Can you see any problem by enabling this?

File: Broadcast-Ext/Broadcast-Ext.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -660,19 +690,23 @@
...
 			buildSettings = {
+				ALLOW_TARGET_PLATFORM_SPECIALIZATION = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				CODE_SIGN_ENTITLEMENTS = "Broadcast-Ext/Broadcast-Ext.entitlements";
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = "";
 				ENABLE_BITCODE = NO;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 i386";
```
Comment (goncalocostamendes) [inline Broadcast-Ext/Broadcast-Ext.xcodeproj/project.pbxproj:709]: @IGitGotIt @jvalli please provide feedback. I was facing the following issue 

> /Users/gmendes/samples/opentok-ios-sdk-samples/Broadcast-Ext/Broadcast-Ext.xcodeproj: error: The linked framework 'Pods_Broadcast_Ext_OpenTok_Live.framework' is missing one or more architectures required by this target: i386. (in target 'OpenTok Live' from project 'Broadcast-Ext')

, that was solved by excluding i386 architecture (arm64 was already there when I added it. Don't know why is not appearing). It does not look very good to me. What is your opinion?

**Review by goncalocostamendes — COMMENTED** [review]
@IGitGotIt @jvalli

**Pull request closed without merging**